Community/kavSimulations/config/KavSim Glareshield Board Config v1-0-0.mfmc

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <ModuleName>Kav Mega Glaresh</ModuleName>
```

Comment:
The board name is truncated and it doesn't contain the new ModuleType information.

Reply:
Kav Glareshield would work.

Reply:
ModuleName changed and ModuleType added